build.gradle

```diff
@@ -85,8 +85,8 @@ task integration(type: Test, description: 'Runs the integration tests.', group:
   classpath = sourceSets.integrationTest.runtimeClasspath
 
   environment("APPINSIGHTS_INSTRUMENTATIONKEY", "test-key")
-  environment("FTP_PRIVATE_KEY", file('src/integrationTest/resources/keypair').text)
-  environment("FTP_PUBLIC_KEY", file('src/integrationTest/resources/keypair.pub').text)
+  environment("FTP_PRIVATE_KEY", file("$projectDir/src/integrationTest/resources/keypair").text)
```

Comment:
file() loads relative to the project directory already, what's the error you get?

Reply:
```
➜  send-letter-service git:(master) ./gradlew clean

FAILURE: Build failed with an exception.

* Where:
Build file '/home/karol/projects/moj/hmcts/send-letter-service/build.gradle' line: 88

* What went wrong:
A problem occurred evaluating root project 'send-letter-service'.
> /home/karol/projects/moj/hmcts/send-letter-service/src/test/resources/keypair (No such file or directory)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
```

Reply:
That error is looking for src/**test** instead of src/**integrationTest**

Reply:
I know.